test/beacon_web/live/page_live_test.exs

```diff
@@ -102,7 +102,6 @@ defmodule BeaconWeb.Live.PageLiveTest do
     test "a given path", %{conn: conn} do
       {:ok, _view, html} = live(conn, "/home")
 
-      assert html =~ "body {cursor: zoom-in;}"
```

Comment:
Removed because we're not running tailwind compiler on test env and that module already has specific tests.